frontend/types/campaigns.ts

```diff
@@ -36,3 +33,63 @@ export type CampaignSearchFormData = {
   endDateFrom: Nullable<Moment>;
   endDateTo: Nullable<Moment>;
 };
+
+export type CampaignListQueryData = {
```

Comment:
```suggestion
export type CampaignListQueryParams = {
```
Nit, I think it's fine to name the query params and body params differently